tests/web/rpc/test_base.py

```diff
@@ -44,6 +44,16 @@ def test_endpoint_name_resolution() -> None:
     assert RPCEndpoint.get_from_name("MyRPC", "v1") is MyRPC
 
 
+def test_list_all_endpoint_names():
+    endpoint_names = RPCEndpoint.list_all_endpoint_names()
+
+    assert "MyRPC" in endpoint_names
+    assert "ErrorRPC" in endpoint_names
+
+    for name in endpoint_names:
+        assert "__" not in name
```

Comment:
are you cropping out the version? That's an important part of the class. There are likely going to be multiple versions of it